x-pack/legacy/plugins/reporting/common/constants.ts

```diff
@@ -27,6 +27,7 @@ export const WHITELISTED_JOB_CONTENT_TYPES = [
   'application/pdf',
   CONTENT_TYPE_CSV,
   'image/png',
+  'text/plain',
```

Comment:
This is needed now so we can pass long certain job-status messages (they were incorrectly being sent as application/json, and I think hapi was fixing that under-the-hood)